sdk/extensions/Microsoft.Azure.WebJobs.Extensions.Clients/src/AzureClientAttribute.cs

```diff
@@ -8,7 +8,7 @@
 namespace Microsoft.Azure.WebJobs
 {
     /// <summary>
-    ///
+    /// Allows binding to an Azure SDK client using connection settings defined in the <see cref="Connection"/> configuration section.
```

Comment:
You may want to add content for the constructor (lines 17-20).  The doc comments there are also empty.

Reply:
Good point